e2e-tests/src/test/validators_change.rs

```diff
@@ -18,9 +19,10 @@ pub fn change_validators(config: &Config) -> anyhow::Result<()> {
     let mut accounts = accounts_from_seeds(seeds);
     let sudo = get_sudo(config);
 
-    let connection = create_connection(node).set_signer(sudo);
+    let connection = RootConnection::new(SignedConnection::new(create_connection(node), sudo));
```

Comment:
this API is not super ergonomic :/ sucks there's no method overloading in Rust, but maybe `new` could take url and sudo keypair, and then have `from_signed_connection` / `from_any_connection` for "type cast constructors" WYT?

Reply:
changed somehow - see whether it works better now